modules/flowable-ui-task/flowable-ui-task-app/src/main/resources/META-INF/flowable-app/flowable-app.properties

```diff
@@ -96,7 +96,7 @@ idm.admin.password=test
 #app.redirect.url.on.authsuccess=http://localhost:8080/flowable-task
 #deployment.api.url=http://localhost:8080/flowable-task/process-api
 
-debugger.enabled=false
+debugger.enabled=true
```

Comment:
Can you change this to false as the default?

Reply:
Don't we want to allow debugging by default? as a feature to show.